widget/edit.html

```diff
@@ -83,7 +83,7 @@ <h3 class="modal-title col-md-10">Edit SOC Management</h3>
                     <span class="text-danger">*</span>
                 </label>
                 <span
-                    data-uib-tooltip="It is recommended to apply filter, that filters out 1 record. If multiple records are filtered with the condition given, first record will be assumed"
+                    data-uib-tooltip="It is recommended to apply filter, that filters out one record. If multiple records are filtered with the condition given, first record will be assumed"
```

Comment:
It is recommended to apply filter, that filters out one record. If multiple records are filtered with the condition given, first record will be assumed --> This widget requires only one JSON record to work hence It is recommended to apply a filter, in case of multiple records are found.